cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/aws-orchestrator/AwsOrchestrator.service.tsx

```diff
@@ -0,0 +1,33 @@
+// React imports
+import React from 'react';
+import { AwsService } from '../../../../../swagger-api';
+import { AWS_FIELDS } from '../../aws-mc-basic/AwsManagementClusterBasic.constants';
+import { INPUT_CHANGE } from '../../../../../state-management/actions/Form.actions';
+import { AwsDefaults } from '../default-service/AwsDefaults.service';
+import { STORE_SECTION_FORM } from '../../../../../state-management/reducers/Form.reducer';
+import { AWS_ADD_RESOURCES } from '../../../../../state-management/actions/Resources.actions';
+import { AwsResourceAction, FormAction, StoreDispatch } from '../../../../../shared/types/types';
+export class AwsOrchestrator extends React.Component {
```

Comment:
I don't think will need React lifecycle hooks so can likely get away with removing extend React.Component. Same for the AwsDefaults.service.tsx

Reply:
Yes sri. You are right. Removed unnecessary import